docs/changelog/1163.md

```diff
@@ -0,0 +1 @@
+- Added the 'support-radius' as an additional configuration option to the Gaussian RBF mapping configuration [#1163](https://github.com/precice/precice/pull/1163).
```

Comment:
Upon common request, the release script now adds the PR number to every changelog entry. So you don't need to do this manually.

```suggestion
- Added the 'support-radius' as an additional configuration option to the Gaussian RBF mapping configuration.
```